docs/README.md

```diff
@@ -92,7 +92,27 @@ See [config.md#instrumented-libraries-and-frameworks](config.md#instrumented-lib
 Download and extract the appropriate binaries from
 [the latest release](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest).
 
-> The path where you put the binaries is referenced as `%InstallationLocation%`
+> The path where you put the binaries is referenced as `$INSTALL_DIR`
+
+You can use a convenient Bash script to do it.
+Parameters:
+
+- `INSTALL_DIR` - location where binaries are to be installed;
+  default: `./otel-dotnet-auto`
+- `FLAVOUR` - possible values: `linux-glibc`, `linux-musl`, `macos`, `windows`;
+  required
+- `RELEASES_URL` - GitHub releases URL;
+  default: `https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases`
+- `TMPDIR` - temporary directory used when downloading the files;
+  default: `$(mktemp -d)`  
+- `VERSION` - version to download;
```

Comment:
nit: I guess that it should only be used for current version and above (just because it won't work for musl for the old ones and I'm not sure if it was tested against the old ones).

Reply:
I'd turn this list into a table with columns `Parameter`, `Required`, and `Description`